include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -233,6 +233,14 @@ struct __atomic_ref : sycl::atomic<_AtomicType, _Space>
 };
 #endif // _ONEDPL_SYCL2023_ATOMIC_REF_PRESENT
 
+template <typename DataT, int Dimensions = 1>
+using __local_accessor =
+#if _ONEDPL_LIBSYCL_VERSION >= 60000
+    sycl::local_accessor<DataT, Dimensions>;
+#else
+    sycl::accessor<DataT, Dimensions, sycl::access::mode::read_write, __dpl_sycl::__target::local>;
```

Comment:
should be 
`sycl::accessor<DataT, Dimensions, std::is_const_v<DataT> ? sycl::access::mode::read :  sycl::access::mode::read_write, __dpl_sycl::__target::local>;`

Reply:
Fixed